sapirbitan/assignment1/ex2.cpp

```diff
@@ -0,0 +1,46 @@
+#include <iostream>
+#include <string>
+using namespace std;
+
+struct some_struct {
```

Comment:
Please give your structs, classes and functions meaningful names. `some_struct` is not a good name because it doesn't describe what the struct holds. Please rename to 'node' or 'listNode'.